cloudpathlib/gs/gsclient.py

```diff
@@ -8,9 +8,13 @@
 from .gspath import GSPath
 
 try:
-    from google.auth.credentials import Credentials
+    from google.auth.credentials import Credentials  # noqa: F401
```

Comment:
If there's a good reason (I'm not familiar with the flake8 rule in particular) to do `"google.auth.credentials.Credentials"` and then flake8 says we don't need this import, then we should just remove it.